packages/rancher-cis-benchmark/charts/values.yaml

```diff
@@ -8,7 +8,7 @@ image:
     tag: v1.0.9
```

Comment:
Seems like we need to update cis-operator tag also for backporting https://github.com/rancher/rancher/issues/38664

Reply:
Ah right! Updated the PR with the latest cis-benchmark rc tag.